src/index.user.js

```diff
@@ -4547,25 +4547,30 @@ function hookUpDailyChallengeAutoCollect() {
     );
     claimAllButtonElement.style.float = "right";
     claimAllButtonElement.style.marginRight = "16px";
-    claimAllButtonElement.textContent = "Claim All";
+    claimAllButtonElement.textContent = "Claim All & Close";
     claimAllButtonElement.addEventListener("click", clickHandler);
     titleRootElement.appendChild(claimAllButtonElement);
 
     function clickHandler() {
       const claimButtonElements = document.querySelectorAll(
         ".missionWrapper:not(.alreadyClaimed) .claimButton, .rewardWrapper:not(.claimed) .claimButton"
       );
+      const closeBtn = document.querySelector('#dailyMissionsPopup > a.close.closeButton[title="Close"]');
```

Comment:
It would be good to stick to a naming variable convention. In our case `closeButtonElement`

Reply:
sure